src/sources/docker_logs.rs

```diff
@@ -297,17 +312,33 @@ impl DockerLogsSource {
         out: Pipeline,
         shutdown: ShutdownSignal,
     ) -> crate::Result<DockerLogsSource> {
+        // Ensure that include_containers and exclude_containers aren't both
+        // specified
+        if config.include_containers.is_some() && config.exclude_containers.is_some() {
+            return Err(Box::new(DockerLogsConfigErrors::IncludeAndExcludeSpecified));
+        }
```

Comment:
Is there a logic reason it doesn't make sense to specify both? I am coming up with cases where you could set, say, some IDs in the excludes and name patterns in the includes or vice versa and have both apply.